package.json

```diff
@@ -36,9 +36,11 @@
     "url": "git://github.com/alexdavid/exprestive.git"
   },
   "scripts": {
+    "compress": "gulp",
     "feature-tests": "./node_modules/.bin/cucumber-js test/feature",
     "prefeature-tests": "gulp",
-    "prepublish": "gulp",
+    "postpublish": "git push && git push --tags",
+    "preversion": "npm test && npm run compress",
```

Comment:
I don't see any reason to compress preversion and I think that should stay as prepublish.